### PR TITLE
feat: wealth endpoint

### PR DESCRIPTION
**Pull request by jyeshe**
## What

Tracks wealth of 100 wallets with biggest balances.

## Why
refs #755 

## Notes
Next there will be: 
- the config of amount of wallets
- consider internal transfers and rewards
- automated tests of async tasks

### CODE REVIEW

File: lib/ae_mdw/sync/server.ex [hunk 1]
```diff
@@ -298,15 +299,29 @@ defmodule AeMdw.Sync.Server do
...
+        |> maybe_enqueue_accounts_balance(block, block_index)
+        |> State.commit_db(block_mutations, clear_mem?)
       end)
 
     broadcast_blocks(gens_mutations)
 
     new_state
   end
 
+  defp maybe_enqueue_accounts_balance(state, _block, {_kbi, -1}) do
```
Comment (sborrazas) [inline lib/ae_mdw/sync/server.ex:313]: Key blocks would affect miner's balances, would it make sense to include them too?

**Review by sborrazas — COMMENTED** [review]
Rest LGTM

File: lib/ae_mdw/sync/server.ex (hunk 1 above)
Comment (jyeshe) [reply lib/ae_mdw/sync/server.ex:313]: Yes, gonna be implemented incrementally. Next there will be these items from the Notes:
- the config of amount of wallets
- consider internal transfers and rewards
- automated tests of async tasks

Comment (sborrazas) [reply lib/ae_mdw/sync/server.ex:313]: Cool, thanks for explaining

**Review by sborrazas — APPROVED** [review]

**Pull request merged**